gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -234,8 +259,8 @@ gst_tensor_query_client_set_property (GObject * object, guint prop_id,
       }
       g_free (self->topic);
       self->topic = g_value_dup_string (value);
-      if (NNS_EDGE_PROTOCOL_TCP == self->protocol)
-        self->protocol = NNS_EDGE_PROTOCOL_MQTT;
+      if (NNS_EDGE_CONNECT_TYPE_TCP == self->connect_type)
```

Comment:
Q: is it necessary to change connect type here?
1. topic is not null and conn-type is tcp, then direct connection.
2. topic is null and conn-type is hybrid, then error. (this needs proper error message when negotiation failure occurs.)

Reply:
Removed! Thanks.